packages/core/src/Stores/ui-store.js

```diff
@@ -485,6 +485,10 @@ export default class UIStore extends BaseStore {
         this.promptFn = cb;
     }
 
+    get is_desktop() {
```

Comment:
Since we dont have any tablet specific design dont we need  a check for values between  MAX_MOBILE_WIDTH & MAX_TABLET_WIDTH `MAX_MOBILE_WIDTH< Screen < MAX_TABLET_WIDTH`. ?

Reply:
note: duplicate exists, remove.